<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>teamdialog</class>
 <widget class="QWidget" name="teamdialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1038</width>
    <height>633</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>-20</x>
     <y>0</y>
     <width>1061</width>
     <height>631</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/dashboard/dashboard.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="addOrEditUser">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>350</y>
     <width>181</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(241, 238, 205);
background-color: rgb(113, 113, 113);
border-radius:6px;</string>
   </property>
   <property name="text">
    <string>Add or Edit User</string>
   </property>
  </widget>
  <widget class="QLabel" name="nameLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 157);
background-color: rgb(241, 238, 205);
border-radius:7px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="descriptionLabel">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>100</y>
     <width>271</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 157);
background-color: rgb(241, 238, 205);
border-radius:7px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="editpro">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>120</y>
     <width>191</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(38, 0, 115);
background-color: rgb(216, 213, 184);
border-radius:6px;</string>
   </property>
   <property name="text">
    <string> Edit Team</string>
   </property>
  </widget>
  <widget class="QPushButton" name="delete_user">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>400</y>
     <width>181</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(241, 238, 205);
background-color: rgb(113, 113, 113);
border-radius:6px;</string>
   </property>
   <property name="text">
    <string>Delete User</string>
   </property>
  </widget>
  <widget class="QLabel" name="idLabel">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>260</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Members  :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="idLineEdit">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>300</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:5px;
background-color: rgb(181, 181, 181);</string>
   </property>
  </widget>
  <widget class="QComboBox" name="roleComboBox">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>300</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:5px;
background-color: rgb(140, 140, 140);</string>
   </property>
   <item>
    <property name="text">
     <string notr="true">Role</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Admin</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>User</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="deletepro">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>160</y>
     <width>191</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(38, 0, 115);
background-color: rgb(216, 213, 184);
border-radius:6px;</string>
   </property>
   <property name="text">
    <string>Delete Team</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
